tests/test_task.py

```diff
@@ -839,6 +840,34 @@ def test_get_environment_values(self):
         self.assertEqual(environ["wsgi.input_terminated"], True)
         self.assertEqual(inst.environ, environ)
 
+    def test_client_disconnect(self):
```

Comment:
Is this test incomplete on purpose?

Reply:
Also a leftover. I first tried to implement the test in `test_task`, but later found it easier to implement it in `test_channel`.